.github/workflows/ci.yaml

```
@@ -0,0 +1,35 @@
+name: Tests
+
+on: pull_request
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+        - 3.6
+        - 3.7
+        - 3.8
+        - 3.9
+        - 3.10-alpha - 3.10.0
```

This is closer to the example at https://github.com/actions/setup-python#usage
```suggestion
        - 3.10.0-alpha - 3.10.0
```